app/src/sharedTest/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityTest.kt

```diff
@@ -148,6 +158,21 @@ class QuestionPlayerActivityTest {
     testCoroutineDispatchers.unregisterIdlingResource()
   }
 
+  @Test
+  fun testQuestionPlayer_hasCorrectActivityLabel() {
+    activityTestRule.launchActivity(
+      createQuestionPlayerActivityIntent(
+        ApplicationProvider.getApplicationContext(),
+        SKILL_ID_LIST as ArrayList<String>
```

Comment:
@Pranav-Bobde the test that you have written is not passing. Please always make sure that you tests pass. It is failing at this line.

Reply:
@rt4914  Done.

Reply:
Ok, tried back with SKILL_ID_LIST  and tried with some reference PRs but still not getting what's the problem here now. Please Help.

Reply:
Have a look at
`launchForSkillList` in this file, you will notice how skills are passed in `QuestionPlayerActivity.createQuestionPlayerActivityIntent` function. Similar to that 

```
    activityTestRule.launchActivity(
      QuestionPlayerActivity.createQuestionPlayerActivityIntent(
        context, ArrayList(SKILL_ID_LIST)
      )
    )
```

Reply:
@rt4914 Done. This doesn't seems to solve the problem too.